behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -191,3 +191,16 @@ def step_impl_resulting_generic_search(context, location):
         And I am taken to the page corresponding to "{location}" result
     """
     )
+
+
+@step('A "{message}" error is returned')
+def step_impl_postcode_not_found_error_returned(context, message):
+    error_title = context.helperfunc.find_by_css_selector(".govuk-error-summary__title")
+    error_message = context.helperfunc.find_by_css_selector(
+        ".govuk-error-summary__body"
+    )
+
+    assert error_title is not None
+    assert error_message is not None
+    assert error_title.text == "There is a problem"
```

Comment:
Can you add this String to the commons.py, just incase more tests are placed in the future that are checking that String.